app/(public)/payment/cancel/page.tsx

```
@@ -0,0 +1,34 @@
+
+    return (
+        <div className="flex min-h-screen items-center justify-center">
+            <div className="text-center">
+                <h1 className="mb-4 text-2xl font-bold text-red-600">Payment Cancelled</h1>
+                <p className="mb-4 text-gray-600">
+                    Your payment was cancelled. You can try again later.
+                </p>
+                <button
+                    onClick={handleGoBack}
+                    className="rounded bg-blue-600 px-4 py-2 text-white hover:bg-blue-700"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)